src/schema/v2/fair.ts

```diff
@@ -199,6 +206,25 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
         // field in Gravity and then update it here.
         resolve: ({ kaws_collection_slugs }) => kaws_collection_slugs,
       },
+      ...(useUnstitchedMarketingCollections && {
+        marketingCollections: {
+          type: new GraphQLList(MarketingCollectionType),
+          args: {
+            size: {
+              type: GraphQLInt,
+              description: "Number of artworks to return",
+            },
+          },
+          resolve: async (
+            { kaws_collection_slugs },
```

Comment:
Surprising that indeed this is a property on a fair! - https://github.com/artsy/gravity/blob/b937ec38e606971cd4011b882abc397082f3968d/app/models/domain/fair.rb#L51